### PR TITLE
Add max execution time to faq

### LINKED ISSUE #292: Add max execution time to FAQs
**What is the expected feature or enhancement?**

Update [Frequently Asked Questions](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/faqs.html) to include [what is the maximum execution time](https://github.com/Qiskit/qiskit-ibm-runtime/blob/stable/0.3/docs/max_time.rst)

**Acceptance criteria**

### PR DESCRIPTION
**Pull request by HuangJunye**
### Summary

Update [Frequently Asked Questions](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/faqs.html) to include [what is the maximum execution time](https://github.com/Qiskit/qiskit-ibm-runtime/blob/stable/0.3/docs/max_time.rst)

image

### Details and comments
Fixes #292

### CODE REVIEW

**Comment by HuangJunye** [comment]
I split faqs into different files. This is how the FAQs page look like.
image

I notice the directory is not setup properly. How do I change to `Docs > FAQs > xxx`?
image

File: docs/faqs/max_execution_time.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+============================================================
```
Comment (rathishcholarajan) [inline docs/faqs/max_execution_time.rst:1]:
```suggestion
.. _faqs/max_execution_time:

============================================================
```

**Comment by rathishcholarajan** [comment]
> I split faqs into different files. This is how the FAQs page look like. image
> 
> I notice the directory is not setup properly. How do I change to `Docs > FAQs > xxx`? image

Adding this seems to fix the breadcrumbs https://github.com/Qiskit/qiskit-ibm-runtime/pull/294/commits/e2afac814844efdcc7e90bcfa6e1c5e7a91b95c6

**Review by rathishcholarajan — APPROVED** [review]
LGTM! Thanks a lot @HuangJunye!

**Pull request merged**